### ISSUE TITLE
Question about the performance drop on the test split

### ISSUE THREAD

**Issue by jianingwangind** [TRI-ML/PF-Track#9]
Hi @ziqipang i noticed that for the full-resolution setting, the amota on val is 0.479 but the one on test is around 0.434. Do you have any ideas why there is this notable drop?

Looking forward to your reply!

**Comment by ziqipang** [comment]
@jianingwangind Such a drop is common for many 3D MOT methods (as in the comparison tables), and my guess is that the test set distribution is more challenging than the validation set. An additional reason is that I tuned the hyper-parameters, such as the length of track extension, on the validation set. The distribution of the test set might fit other hyper-parameters.

**Comment by jianingwangind** [comment]
Thanks a lot for you reply.

**Comment by jianingwangind** [comment]
@ziqipang Hi may i also ask that why you choose different training settings for 800x320 and 1600x640? have you also tried to train 1600x640 with the training configs for 800x320? If yes, how does the result look like? 
Thanks  a lot.

**Comment by ziqipang** [comment]
@jianingwangind The reason was actually simple. I didn't have GPUs that have sufficiently large GPU memories to enable the training of backbones under the full-resolution setting. Therefore, I was unable to try it, but I encourage you to try it.

As for your second question, 
> In my case, i have complemeted several experiments with 800x320 which improve amota; but when i switch to 1600x640 with the rest experiment settings unchanged, amota doesn't improve much,
Just to clarify, do you mean several measures that work well on 800x320 are not so effective for 1600x640?

**Comment by jianingwangind** [comment]
@ziqipang Thanks for your reply. For the second question yes, at least based on my current experiments it appears so, so strange

**Comment by ziqipang** [comment]
@jianingwangind Same as you, I don't have too many ideas on this, because I haven't fully experimented on this. Just a few intuitions, if they are helpful.
1. Optimization. Moving to a larger resolution might make the optimization more difficult. Perhaps you can use more training epochs.
2. Data distribution. Using a larger resolution could improve the detection performance, especially on small objects. Therefore, you might need to change some configurations, e.g., the number of queries, the score threshold, etc.

**Comment by jianingwangind** [comment]
@ziqipang Thanks a lot for your help:) I will definitely try these.
